xrpl_api/src/api/account_offers.rs

```diff
@@ -66,6 +68,31 @@ pub struct AccountOffer {
     pub taker_pays: Amount,
 }
 
+impl PartialEq for AccountOffer {
```

Comment:
This is a weird implementation of equality. I am not sure it should stay in this file.

Reply:
Yes, this seems like very application specific. I think it is better not to add it in the first place? The logic should be in `xrpl_cli` crate.

Reply:
However, ordering by sequence _is_ 'natural' for AccountOffers, no?

Reply:
Even if only temporary, I think it is important not to have it here in `xrpl_api` and `xrpl_types` even temporarily, because it can create unexpected result for users. If `PartialEq` and `Ord` are implemented they should behave as "expected", not a context specific behaviour.

Reply:
You can create a newtype wrapper (in `xrpl_cli`) if you want a type like `AccountOffer` with a specific ordering or sort them using a sorting method.

Reply:
Ok, I will remove.

Reply:
> However, ordering by sequence _is_ 'natural' for AccountOffers, no?

Only in the specific context where you are looking at offers for a specific account (api endpoint gives you for a specific account, but nothing prevents you from mixing values from different calls to the endpoint with different accounts). And even for a single account, you could also argue by ordering by price or amount.